snuba_sdk/query_visitors.py

```diff
@@ -396,12 +397,23 @@ def _combine(self, query: "main.Query", returns: Mapping[str, None]) -> None:
             raise InvalidQuery("query must have at least one expression in select")
 
         # Non-aggregate expressions must be in the groupby if there is an aggregate
-        non_aggregates = []
+        non_aggregates: List[Union[Column, CurriedFunction, Function]] = []
```

Comment:
Is this change related with this warning `"groupby must be included if there are aggregations in the select" `  from prod ?

Reply:
Yes.